libnavigation-core/src/main/java/com/mapbox/navigation/core/replay/route2/ReplayRouteOptions.kt

```diff
@@ -5,34 +5,32 @@ package com.mapbox.navigation.core.replay.route2
  * allow you to control the behavior of that driver.
  *
  * Note that the default values are recommended because they have been tested.
+ *
+ * @param maxSpeedMps Max speed the driver will drive on straight-aways
+ * @param turnSpeedMps Speed the driver will slow down for turns approaching 90 degrees
+ * @param uTurnSpeedMps Speed the driver will go when facing a u-turn
+ * @param maxAcceleration How fast the driver will accelerate to [maxSpeedMps] in mps^2
+ * @param minAcceleration How fast the driver will decelerate in mps^2
+ * @param builder used for updating options
  */
 class ReplayRouteOptions(
-    /**
-     * Max speed the driver will drive on straight-aways
-     */
     val maxSpeedMps: Double,
-    /**
-     * Speed the driver will slow down for turns approaching 90 degrees
-     */
     val turnSpeedMps: Double,
-    /**
-     * Speed the driver will go when facing a u-turn
-     */
     val uTurnSpeedMps: Double,
-    /**
-     * How fast the driver will accelerate to [maxSpeedMps] in mps^2
-     */
     val maxAcceleration: Double,
+    val minAcceleration: Double,
+    val builder: Builder
+) {
     /**
-     * How fast the driver will decelerate in mps^2
+     * @return the builder that created the [ReplayRouteOptions]
      */
-    val minAcceleration: Double
-) {
+    fun toBuilder() = builder
+
     /**
      * Used to build [ReplayRouteOptions].
      */
     class Builder {
-        private var maxSpeedMps = 30.0
+        private var maxSpeedMps = 20.0
```

Comment:
🔊 📢 

Slipping a default change in this pull request. There were multiple mentions that the simulated driver is too fast. Also, we have the ability to speed up the replay speed using MapboxReplayer.playbackSpeed

Reply:
failed tests 🤔  going to look into it separately